recipes/openconnect-sso/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "openconnect-sso" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/vlaci/openconnect-sso/releases/download/v{{ version }}/openconnect-sso-{{ version }}.tar.gz
+  sha256: a7bde0a8dd35d7bab3c66f28b420c343c270a7ee45e98e04bfa6ecc6958f5c5b
+
+build:
+  skip: True  # [not unix]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+  entry_points:
+    - openconnect-sso = openconnect_sso.cli:main
+
+requirements:
+  build:
+    - poetry >=0.12
```

Comment:
Why do you need poetry as a build and host dependency?

Reply:
Removed the build section in 4751cd4.